src/main/kotlin/org/rust/lang/core/stubs/RsPlaceholderStub.kt

```diff
@@ -9,24 +9,25 @@ import com.intellij.lang.ASTNode
 import com.intellij.psi.stubs.*
 import org.rust.lang.core.psi.ext.RsElement
 
-open class RsPlaceholderStub(parent: StubElement<*>?, elementType: IStubElementType<*, *>)
-    : StubBase<RsElement>(parent, elementType) {
+open class RsPlaceholderStub<PsiT : RsElement>(parent: StubElement<*>?, elementType: IStubElementType<*, *>)
```

Comment:
@vlad20012 could you take a look at this change, please? maybe you have a better idea how to fix the type inference in `org.rust.lang.core.stubs.RsMetaItemStub#getValue` property body

Reply:
LGTM